ssp

```diff
@@ -2505,8 +2514,28 @@ sub print_ftpserver_info {
 
     my $passivetext = $passiveports ? "enabled - " . ( $fwppactive ? "allowed in iptables" : "not found in iptables" ) : "not enabled";
 
+    my $passiveiptext = "";
+
+    if ($passiveip) {
+        if ( $passiveip ne $EXTERNAL_IP_ADDRESS ) {
+            if ( defined( $CPCONF{'ftpserver'} ) ) {
```

Comment:
These if blocks should be combined:
if ( defined( $CPCONF{'ftpserver'} ) and ( $passiveip ne $EXTERNAL_IP_ADDRESS ) ) {